klaytn-api/app.js

```diff
@@ -23,51 +24,72 @@ const dpt = new devp2p.DPT(Buffer.from(PRIVATE_KEY, 'hex'), {
 	address: '0.0.0.0',
 	udpPort: null,
 	tcpPort: null,
+    type: 3,
     },
 });
 
 var nodeList = {};
 var nodeCount = 0;
 dpt.on('peer:added', (peer) => {
     if (peer.address in nodeList) {
-	return
+        console.info("node count", nodeCount)
+	    return
     }
-    checkRPC(peer.address, (addr) => {
-	nodeList[addr] = new nodeModel(addr, '8551');
-	nodeCount++;
-	console.log("peer added: ", addr);
+    checkRPC(peer, (peer, rpcPort, onRpc) => {
+        nodeCount++;
+        if(onRpc) {
```

Comment:
```suggestion
        if(onRpc) {
                nodeCount++;
```
Should we increment `nodeCount` when `onRpc` is false?